Source/JavaScriptCore/runtime/MathObject.cpp

```diff
@@ -246,27 +246,35 @@ JSC_DEFINE_HOST_FUNCTION(mathProtoFuncHypot, (JSGlobalObject* globalObject, Call
     }, NulloptBehavior::Abort);
     RETURN_IF_EXCEPTION(scope, { });
 
-    double max = 0;
+    bool hasNaN = false;
+    double maxAbs = 0;
+    double sum = 0;
     for (double argument : args) {
```

Comment:
If we wanted to make this faster, I would expect it's faster to do the `toNumber()` inside this loop. Nothing in this loop has observable effects and throwing is likely rare. That avoids the vector entirely.